packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,379 @@
+
+We currently have 3 account-level areas:
+
+1. **Account:** Accessed from the masthead.
+2. **Billing:** Accessed from the masthead.
+3. **Support Center:** Accessed from Docs &amp; support menu in left nav.
+
+If there is a need to access a page in an account-level area from your product, such as API Keys or Credentials,
+link to the page from your product rather than duplicating the page inside of your product.
+This eliminates the need to duplicate content in multiple places.
+
+## Vertical sidebar navigation vs in-page horizontal tabs
+
+The left sidebar navigation should include all the highest-level primitives related to your product.
+Within a given page, horizontal tabs can be used to segment different categories of information related to
+that primitive, or different variations of that primitive.
+
+## When to use breadcrumbs
+
+Breadcrumbs are used sparingly. A breadcrumb is only visible when viewing a page that is secondary or tertiary
+to the page selected in the main navigation. The breadcrumb gives you the ability to link back to the primary page.
+It also helps to remind you where you are in the hierarchy of pages.
+
+If you were to navigate to a tertiary page, the breadcrumb would show both the primary and secondary breadcrumbs
+on the tertiary page ( Logs / Log detail / ). The first breadcrumb will always match the page
+highlighted in the nav.
+
+![](../../../assets/images/experiences/console/breadcrumb-example.png)
+
+<ImageCaption>A screenshot of breadcrumbs being used on a Console page</ImageCaption>
+
+---
+
+## Hiding and showing pages in the navigation
+
+When a user views or pins a product to the navigation, all pages should be immediately visible.
+Pages should not be progressively shown based on user state or usage. Instead, the content of the page should
+evolve to help the user understand the feature and/or start using it. You can use an [empty state](/patterns/empty-state) with an
+illustration and information about the feature.
+
+Hiding a page in the nav (via an account flag) should be done only when you are limiting access to that feature.
+For example, the feature is in private beta or the feature is deprecated and supported only for existing users.
+Hiding and showing pages can also be done when you’re limiting access based on RBAC.
+
+## Beta tags
+
+Beta should appear in the nav under the product name when a product is in beta. It can also appear under a
+single page in a product nav when just that feature is in beta. In addition, you should add the beta badge
+next to the H1 in the page for the feature in beta. In the case where the entire product is in beta,
+include the badge next to the H1 on the first page in the nav. The “beta” is clickable and opens a modal
+describing what beta means.
+
+![](../../../assets/images/experiences/console/beta-tag.png)
+
+<ImageCaption>A beta tag being used in Console to identify and describe a beta-stage product.</ImageCaption>
```

Comment:
Suggestion(non-blocking): kinda similar to the other image, could this be more descriptive?

> A beta tag being used in the header of a Console page to identify and describe a beta-stage product.